aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -79,6 +79,7 @@ module aptos_framework::coin {
     /// Represents a coin with aggregator as its value. This allows to update
     /// the coin in every transaction avoiding read-modify-write conflicts. Only
     /// used for gas fees distribution by Aptos Framework (0x1).
+    #[deprecated]
```

Comment:
We use it in `CollectedFeesPerBlock` which has never been used. Should that also be deprecated?

Reply:
I can mark that as deprecated too

Reply:
You should probably do that.  While we still suppress warnings on Aptos framework code, that's kind of kludgey.  We also suppress use of deprecated things warnings in code which is itself deprecated, which is less kludgey.